antora.yml

```diff
@@ -1,7 +1,57 @@
 name: docs
 title: Documentation
-version: master
+version: main
 nav:
   - modules/ROOT/nav.adoc
 
 start_ROOT: ROOT:index.adoc
+
+asciidoc:
+  attributes:
```

Comment:
We need to change these attributes to match our downstream values. They are in the comments.